src/favorites/FavoriteCategoryButton.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import Icon from '../widgets/Icon';
 
-const FavoriteCategoryButton = ({ name, onClick, isFavorited }) => {
+const FavoriteCategoryButton = ({ name, onClick, isFavorited, noTitle }) => {
   return (
-    <h2 className="mt-1 mb-0 text-xs-center">
```

Comment:
Abviously this would need some small change here too: https://github.com/adcpm/busy/blob/38fa1adb8fd4f7e13cd9a3dbfaad3092466d7ff6/src/feed/Page.js#L92